packages/react-components/lib/robots/robot-info.tsx

```diff
@@ -55,6 +69,11 @@ export function RobotInfo({
     <StyledDiv>
       <Typography variant="h6" style={{ textAlign: 'center' }} gutterBottom>
         {robotName}
+        <Typography variant="caption" display="block">
+          {updateTime !== undefined
+            ? `Last update - ${calculateLastUpdateTime(updateTime).toLocaleString()}s ago`
```

Comment:
This part is interesting, I couldn't find what `String.toLocaleString` do. If that is required to get proper string for display, should we be applying this to all UI elements?